File: aliyun-python-best-practise/为ECS创建数据盘并挂载到服务器上/createdisk_demo.py
from aliyunsdkcore.client import AcsClient
from aliyunsdkecs.request.v20140526.CreateDiskRequest import CreateDiskRequest
client = AcsClient('access_key_id', 'access_key_secret', 'region_id')
request = CreateDiskRequest()
request.set_ZoneId('cn-hangzhou-e')  # 设置云盘所在区域
request.set_Size(40)  # 设置云盘大小
request.set_DiskCategory("cloud_efficiency")  # 设置云盘种类
response = client.do_action_with_exception(request)
print(response)

File: aliyun-java-best-practise/src/main/java/com/aliyun/bestpractise/CreateDiskDemo.java
package com.aliyun.bestpractise;

import com.aliyuncs.DefaultAcsClient;
import com.aliyuncs.IAcsClient;
import com.aliyuncs.ecs.model.v20140526.CreateDiskRequest;
import com.aliyuncs.ecs.model.v20140526.CreateDiskResponse;
import com.aliyuncs.exceptions.ClientException;
import com.aliyuncs.profile.DefaultProfile;

public class CreateDiskDemo {
    public static void main(String[] args) {
        DefaultProfile profile = DefaultProfile.getProfile(
                "RegionId",
                "accessKeyId",
                "accessKeySecret");
        IAcsClient client = new DefaultAcsClient(profile);
        CreateDiskRequest request = new CreateDiskRequest();
        //设置云盘所在区域
        request.setZoneId("cn-hangzhou-e");
        //设置云盘大小
        request.setSize(40);
        //设置云盘种类
        request.setDiskCategory("cloud_efficiency");
        CreateDiskResponse response;
        try {
            response =  client.getAcsResponse(request);
        } catch (ClientException e) {
            e.printStackTrace();
        }
    }
}
